src/snowflake/snowpark/modin/plugin/docstrings/window.py

```diff
@@ -142,31 +139,27 @@ def sem():
         2  2.0
         3  NaN
         4  4.0
-        dtype: float64
         >>> df.rolling(2, min_periods=1).sum()
              B
         0  0.0
         1  1.0
         2  3.0
         3  2.0
         4  4.0
-        dtype: float64
         >>> df.rolling(2, min_periods=2).sum()
              B
         0  NaN
         1  1.0
         2  3.0
         3  NaN
         4  NaN
-        dtype: float64
         >>> df.rolling(3, min_periods=1, center=True).sum()
              B
         0  1.0
         1  3.0
         2  3.0
         3  6.0
-        4  4.0
-        dtype: float64"""
+        4  4.0"""
```

Comment:
do you know why no dtype is displayed? Is it because what we returned is not a series?

Reply:
a DataFrame is returned, not a Series. I'll add a test for that

Reply:
Is this a bug? @sfc-gh-nkrishna

Reply:
I don't believe so, in both pandas and Snowpark pandas I don't see a dtype. Both return a dataframe

Reply:
ok, that's a "bug" in the previous docstring then. I wound not trust the current docstrings so please make sure we update them with the right ones.